{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly.
+
+### PR Cover Letter
+- [ ] Include A detailed description of the changes in the body of the PR
+
+### Testing
+- [ ] Unit, integration, acceptance tests as appropriate
+- [ ] Database migrations are backwards-compatible
+
```

Comment:
I like to have a section for manual testing:

``[ ] Changes have been manually tested as appropriate.``

In fact, it would be better if this was a section that the developer should fill in describing the manual testing that they have done (if appropriate).